### DISCUSSION TITLE
OpenMoji-Support

### DISCUSSION THREAD

**Discussion by linux-lukas** [luizdepra/hugo-coder#694] [Ideas]
I think it might be useful to focus on the free and open source OpenMoji project for icons: https://github.com/hfg-gmuend/openmoji

What do you think about this, @luizdepra ?

**Comment by luizdepra** [comment]
We used `Tweemoji`[https://twemoji.twitter.com/] in the past. But now Hugo has a native [emoji support](https://gohugo.io/getting-started/configuration/#enableemoji), so we are using it.

Of course, you could adapt the theme to use OpenMoji. However, I don't see any benefit by adding this new dependency if Hugo already solve this problem for us.
